addons/web/controllers/main.py

```diff
@@ -1030,10 +1030,24 @@ def test_mobile_suite(self, mod=None, **kwargs):
     def benchmarks(self, mod=None, **kwargs):
         return request.render('web.benchmark_suite')
 
+    @http.route('/web/bundle/<string:bundle_name>/<string:jscss>', type='http', methods=["get"], auth="public", cors="*")
+    def bundle(self, bundle_name, jscss):
+        js = True if "js" in jscss else False
+        css= True if "css" in jscss else False
```

Comment:
What is the point of this? It seems completely unused, with `jscss` always being `"jscss"`

Also, the ternary is useless and so are the locals.

Reply:
Thanks for the review :)

Don't say the ternaries are useless, it hurts their feelings... But yes in this case we always request both so I will remove them :)